File: IntroToComputerScience/Java/FtoC.java
/*
    @author: Ben Rodriguez
    @description: Simple temperature converter F to C
*/

public class FtoC
{
    public static void main(String[] args)
    {
        double farenheit = 40;
        double celsius = ((farenheit - 32) / 1.8);
        System.out.println(celsius);
    }
}

File: IntroToComputerScience/C++/FtoC.cc
/*
    @author: Ben Rodriguez
    @description: Simple temperature converter from F to C
*/

#import <iostream> 

int main()
{
    double farenheit = 40; // farenheit 
    double celsius = ((farenheit - 32) / 1.8); // conversion formula
    std::cout << celsius << std::endl; // print
}